Assets/Scripts/Lua/LuaScriptBinder.cs

```diff
@@ -140,7 +140,8 @@ public static class LuaScriptBinder {
     public static string GetState() {
         try {
             return (UIController.instance.frozenState != UIController.UIState.PAUSE) ? UIController.instance.frozenState.ToString() : UIController.instance.state.ToString();
-        } catch {
+            }
+        catch {
```

Comment:
Why did you change that? The suggestion below impacts the lines 143 and 144:

```suggestion
        } catch {
```